### PR TITLE
Crazy cataclysm fixes (bear/mr skeltal spawns, bionic)

### LINKED ISSUE #26036: DEBUG: warnings for type bio_chest_gun on Character Creation
**Describe the bug**  
DEBUG    : warnings for type bio_chest_gun:
there is no bionic with id bio_chest_gun
 
 FUNCTION : void Item_factory::check_definitions() const
 FILE     : src/item_factory.cpp
 LINE     : 1055

**To Reproduce**  
Fresh install of PR 8007 with Loader added to folder. Created World using mod settings below 
and attempting to Create Character. Error appears when 'Items' is being loaded when using Create menu.

**Expected behavior**  
Issue can be (I)gnored and does not return when creating another character but game experiences slowdown when Sleeping or crafting. CTD happens once game freeze happens and .exe remains in system memory until PC restarts. (after force-quit with TaskManager) Memory error?

Error returns when game is Quit and restarted. Error text does not show if creating multiple characters in a single session.

**Screenshots**  

**Versions and configuration(please complete the following information):**
If any additional information is needed, please request.

**Additional context**  
MOD LIST:
  "dda",
  "craftgp",
  "safeautodoc",
  "brightnights",
  "makeshift",
  "More_Survival_Tools",
  "modular_turrets",
  "Salvaged_Robots",
  "crazy_cataclysm",
  "alt_map_key",
  "Mining_Mod",
  "boats",
  "deoxymod",
  "necromancy",
  "no_rivtech_guns",
  "no_olg_guns",
  "no_flaming_weapons",
  "no_medieval_items",
  "more_classes_scenarios",
  "manualbionicinstall",
  "sleepdeprivation",
  "StatsThroughSkills"

Savegame included.
[Crazy Test 2.0.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/2443635/Crazy.Test.2.0.zip)

### PR DESCRIPTION
**Pull request by Soyweiser**
#### Summary
SUMMARY: Bugfixes "Smokey bear and Mr skeltal spawns updated, fixed bionic problem."

#### Purpose of change
Noticed some errors with crazycataclysm, first that some of the monsters are not defined in the mod, but in the main monsterlist (which is big exception, compared to other mods). Then there also was the bionic ID warning on load. This is an attempt at fixing it (which failed for 3 monsters, see below).

#### Describe the solution
Moved Smokey bear to crazy_monsters.json.
Created CrazyCataclysm/crazy_monstergroups.json and moved Smokey bear spawns to there, also added spawns for Mr Skeletal (which were missing). Smokey is more likely to spawn in caves during winter, and Mr skeletal is more likely during autumn (due to halloween).
Added missing bionic definition for bio_chest_gun. Fixes #26036 but not #25935 (which is caused by the obsolete mod).

#### Describe alternatives you've considered
Currently the "mon_zombie_dancer", "mon_zombie_jackson", "mon_shia" monsters are hardcoded in mapextras.cpp and mapgen_functions.cpp (mon_zombie_jackson doesn't even seem to have a related map special effect and cannot show up at all), I'm not sure if I could move them without problems. So these monsters are still in the main monsters.json file. (I probably can move them if I also move the chance of overmap specials to the crazycata mod, but have not messed with that yet, as I think it isn't possible to just say 'add these overmap specials to the list' without copying the whole list).

#### Additional context
Tested with a world with crazy cata enabled, and a world without to check for errors.

### CODE REVIEW

**Comment by Xhuis** [comment]
I'm a tad surprised that Mr. Skeltal spawns were bugged, because I recall testing them. Perhaps I forgot to commit the spawning code, or include the file.

**Comment by Soyweiser** [comment]
It happens, pretty easy to forget adding a random file.

File: data/mods/CrazyCataclysm/crazy_monstergroups.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[
```
Comment (ZhilkinSerg) [inline data/mods/CrazyCataclysm/crazy_monstergroups.json]: File should be linted.

File: data/mods/CrazyCataclysm/modinfo.json [hunk 2]
```diff
@@ -10,6 +10,6 @@
   },
```
Comment (ZhilkinSerg) [inline data/mods/CrazyCataclysm/modinfo.json]: File should be linted too.

**Pull request merged**